scripts/build/package.js

```diff
@@ -76,11 +80,28 @@ data.icons.forEach(icon => {
   icon.slug = filename;
   icons.push(icon);
 
+  const iconObject = iconToObject(icon)
+
   // write the static .js file for the icon
   const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
-  minifyAndWrite(jsFilepath, `module.exports=${iconToObject(icon)};`);
+  minifyAndWrite(jsFilepath, `module.exports=${iconObject};`);
+
+  // add export to the barrel file
+  const firstLetter = icon.slug[0].toUpperCase();
+  const rest = icon.slug.slice(1);
+  const iconExportName = `icon${firstLetter}${rest}`;
+
+  iconsBarrelJs.push(`${iconExportName}: ${iconObject},`);
+  iconsBarrelMjs.push(`export const ${iconExportName} = ${iconObject}`);
```

Comment:
Minor but do you think you can refactor this to something that looks like

```suggestion
  const iconExportName = slugToIconExportName(icon.slug);
  iconsBarrelJs.push(`${iconExportName}:${iconObject},`);
  iconsBarrelMjs.push(`export const ${iconExportName}=${iconObject}`);
```

Where `slugToIconExportName` is something like:

```js
function slugToIconExportName(slug) {
  const slugFirstLetter = icon.slug[0].toUpperCase();
  const slugRest = icon.slug.slice(1);
  const iconExportName = `icon${slugFirstLetter}${slugRest}`;
}
```